Lombiq.HelpfulExtensions/Extensions/OrchardRecipeMigration/Services/OrchardExportToRecipeConverter.cs

```diff
@@ -67,15 +78,18 @@ await _contentConverters
             }
         }
 
-        foreach (var converter in _exportConverters)
-        {
-            await converter.UpdateContentItemsAsync(export, contentItems);
-        }
+        var urlHelper = _urlHelperFactory.GetUrlHelper(_actionContextAccessor.ActionContext!);
 
-        var recipe = JObject.FromObject(new RecipeDescriptor())!;
-        recipe["steps"] = new JsonArray(JObject.FromObject(new { name = "content", data = contentItems }));
+        bool hasNextPage = page < totalPages;
+        string nextUrl = hasNextPage ? urlHelper.Action(nameof(Convert), "OrchardRecipeMigrationAdmin", new { page = page + 1 }) : null;
```

Comment:
You can do `typeof(OrchardRecipeMigrationAdminController).ControllerName()` instead of the `"OrchardRecipeMigrationAdmin"` string.